packages/cli/docs-resolver/src/ApiReferenceNodeConverterLatest.ts

```diff
@@ -272,29 +271,28 @@ export class ApiReferenceNodeConverterLatest {
 
         const nodeId = this.#idgen.get(overviewPageId ?? maybeFullSlug ?? parentSlug.get());
 
-        const subpackageIds = section.referencedSubpackages
+        const subPackageTuples = section.referencedSubpackages
             .map((locator) => {
                 const subpackage = this.#apiDefinitionHolder.getSubpackageByLocator(locator);
-
-                return subpackage != null ? subpackage.id : undefined;
-            })
-            .filter((subpackageId) => {
-                if (subpackageId == null) {
-                    this.taskContext.logger.error(`Subpackage ${subpackageId} not found in ${this.apiDefinitionId}`);
+                if (subpackage === null || subpackage === undefined) {
+                    this.taskContext.logger.error(
+                        `Unable to add subsection ${locator} to section ${section.title} due to error: ` +
```

Comment:
Let's update the message to: `Failed to locate API section plantInfo. Did you mean [a], [b] or [c]` ?